### ISSUE TITLE
Camera permanently running needlessly drains battery

### ISSUE THREAD

**Issue by heckert-gamedev** [pretix/pretixscan-android#109]
I had the app running on my phone for an event last weekend when there was no queue but only occasional scans required. Keeping it running continuously would have drained the battery a lot with the camera being active all the time.
My workaround was to close the app in between, but it would have been very useful to have a mode where a scan (and thus activation of the camera) would only be done on pushing a "scan now" button.
Ideally one would have some slider or similar toggle to switch between "one off" scans and "continuously scanning" so if some larger group arrives at a time one won't need to press the scan button again and again in this case...

**Comment by raphaelm** [comment]
I honestly think the workaround of "lock the phone when you don't need it" is the best, otherwise the screen will still draw a lot of power.

**Comment by heckert-gamedev** [comment]
Does locking the phone switch off the camera? How can one be sure of it?
Also I think just having the screen on alone would be far less of a drain than also having the camera active and the app trying to scan for some QR code within the image it reads from the camera...

**Comment by raphaelm** [comment]
Yes, locking the phone switches off the camera. I'm not sure how you could verify.
